### DISCUSSION TITLE
How to Run Certain Collections on Daily

### DISCUSSION THREAD

**Discussion by AwesomeAustn** [meisnate12/Plex-Meta-Manager#162] [Q&A]
I'm on unRAID and I'm wondering how to run certain collections on the daily run. This didn't work:

![Screenshot_2021-04-03 unRAID UpdateContainer](https://user-images.githubusercontent.com/6405444/113485813-50501c80-9464-11eb-98e5-a4f3f07bf37e.png)

**Comment by meisnate12** [comment]
what was the output when you ran this?

**Reply by AwesomeAustn** [reply]
The daily log showed it ran all collections.

**Reply by AwesomeAustn** [reply]
It runs fine manually with:
`python plex_meta_manager.py -r -cl "IMDb Top 250, TMDb Top Rated"`
or
`python plex_meta_manager.py -r --collections "IMDb Top 250, TMDb Top Rated"`

**Reply by meisnate12** [reply]
maybe try adding the quotes in the value box?

**Reply by AwesomeAustn** [reply]
Ok, I'll try it. I just didn't think I needed them because `time` `divider`, and `width` don't have them.

**Reply by meisnate12** [reply]
you usually only need them if theres a space involved

**Reply by AwesomeAustn** [reply]
Ah ok

**Reply by AwesomeAustn** [reply]
I can't seem to trigger the daily by changing the time. It ran at 3:00 when it was set to `6:00`. I don't remember if I had `6:00` or `06:00`. I tried to have it run at 12:00 and it didn't run. Since it ran at 03:00 last night instead of of 06:00, I tried `15:10` to try to trigger it for 12:10, and it didn't work either.

**Reply by AwesomeAustn** [reply]
I just found the Schedule Attribute page in the Wiki. I'll try that.

**Reply by AwesomeAustn** [reply]
The Schedule Attribute worked for me for what I was trying to do.

**Reply by meisnate12** [reply]
glad you got it working check out the discord for a better response time in messages
